test/e2e/reclaim.go

```diff
@@ -7,9 +7,178 @@ import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	vcbatch "volcano.sh/volcano/pkg/apis/batch/v1alpha1"
+	schedulingv1beta1 "volcano.sh/volcano/pkg/apis/scheduling/v1beta1"
 )
 
 var _ = Describe("Queue E2E Test", func() {
+	It("Reclaim: New queue with job created no reclaim when resource is enough", func() {
+		q1 := "default"
+		q2 := "reclaim-q2"
+		ctx := initTestContext(options{
+			queues:             []string{q2},
+			nodesNumLimit:      4,
+			nodesResourceLimit: CPU1Mem1,
```

Comment:
nit: How could you know how much resource and how many nodes to occupy?

Reply:
According to test cases, and there is a node num check in init processes, the check will tell user to skip some cases that they can not afford.